### PR TITLE
add test for videobox component

### LINKED ISSUE #127: (SP: 1) Write unit test for "VideoBox" component
### Component unit test
Unit test for "**VideoBox**" component
Scenaries descriptions:
- [] Video should be in the component

[Link to component](https://github.com/Space2Study-Project/Space2Study-FrontEnd/blob/develop/src/components/video-box/VideoBox.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by LeonidShamarin**
Unit test for "VideoBox" component
Снимок экрана (2811)1
Снимок экрана (2812)1

### CODE REVIEW

File: tests/unit/components/video-box/VideoBox.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
```
Comment (vlad-khrychov) [inline tests/unit/components/video-box/VideoBox.spec.jsx]: Can we delete this import?
```suggestion
```

**Review by marianamartsy — APPROVED** [review]
👍

**Review by Mygal-Alex — APPROVED** [review]
test looks good

**Review by AndriiNasadiuk — CHANGES_REQUESTED** [review]
Rows 10 and 12 don’t have influence on coverage, so i would delete it

**Comment by AndriiNasadiuk** [comment]
Знімок екрана 2023-12-07 о 20 51 07

**Comment by LeonidShamarin** [comment]
> Rows 10 and 12 don’t have influence on coverage, so i would delete it

ok

**Comment by AndriiNasadiuk** [comment]
> > Rows 10 and 12 don’t have influence on coverage, so i would delete it
> 
> ok

so "ok", but that rows still in the file))

**Review by AndriiNasadiuk — APPROVED** [review]
nice

**Review by ArtemHolikov — APPROVED** [review]
👍

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**